docs/versionIndex.md

```diff
@@ -105,7 +112,13 @@ backend services have been tested with each released version of ADF.
 
 <!--v510 end-->
 
-## v4.7.0
+## v5.0.0
+
+<!--v500 start-->
+
+<!--v500 end-->
+
+## v5.0.0
```

Comment:
There is a typo here. You've replaced 4.7.0 with 5.0.0, removing that entry and breaking the links to that version. Also, there are 2 entries for 5.0.0 now, one empty and one with the Properties viewer widget

Reply:
changed.